bodhi/client/__init__.py

```diff
@@ -370,11 +370,25 @@ def edit(user, password, url, **kwargs):
             resp = client.query(**query_param)
             title = resp['updates'][0]['title']
         elif re.search(bindings.UPDATE_TITLE_RE, kwargs['update']):
+            query_param = {'like': kwargs['update']}
+            resp = client.query(**query_param)
             title = kwargs['update']
         del(kwargs['update'])
         kwargs['builds'] = title
         kwargs['edited'] = title
 
+        data = munch.unmunchify(resp)
```

Comment:
you should be able to use the Munch object just like a dictionary, so I am not sure this is needed